providers/google/src/airflow/providers/google/cloud/operators/translate.py

```diff
@@ -641,9 +644,21 @@ def __init__(
         self.retry = retry
         self.gcp_conn_id = gcp_conn_id
         self.impersonation_chain = impersonation_chain
+        self.raise_for_empty_result = raise_for_empty_result
 
     def execute(self, context: Context):
         hook = TranslateHook(gcp_conn_id=self.gcp_conn_id, impersonation_chain=self.impersonation_chain)
+        initial_dataset_size = self._get_number_of_ds_items(
+            dataset=hook.get_dataset(
+                dataset_id=self.dataset_id,
+                project_id=self.project_id,
+                location=self.location,
+                retry=self.retry,
+                timeout=self.timeout,
+                metadata=self.metadata,
+            ),
+            total_key_name="example_count",
```

Comment:
@olegkachur-e  I think you can default example_count in _get_number_of_ds_items. If someone will need to change it in future there will be only one place where to change. Or maybe you have other thoughts on it?